slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -156,17 +156,34 @@ class SlideToActView @JvmOverloads constructor(
         }
 
     /** Custom Slider Icon */
-    @DrawableRes
-    var sliderIcon: Int = R.drawable.slidetoact_ic_arrow
+    var sliderIcon: Any = R.drawable.slidetoact_ic_arrow
```

Comment:
This is really suboptimal :/

We're now exposing a public property which is a `Any`.
I think we should instead create other two private fields called:

```
private var sliderIconDrawable
private var completeIconDrawable
```

And let the BindingAdapter operate on those.